golang/pkg/rnr/task_nested.go

```diff
@@ -19,8 +21,9 @@ type NestedTask struct {
 }
 
 type NestedTaskOptions struct {
-	Parallelism int  // the number of tasks to run in parallel; defaults to 1
-	CompleteAll bool // if `true`, the NestedTask will attempt to run all tasks before transitioning to either SUCCEEDED or FAILED state.
+	Poll        NestedTaskCallback // a callback called each time a Poll() on NestedTask is called.
```

Comment:
I found this name and the type declared above a bit confusing.

Regarding the type `NestedTaskCallback` it doesn't seem to be used as a callback but rather a custom poll function… which leads me to think that the option field should instead be named `CustomPoll` or similar instead. WDYT?

Reply:
Agreed, makes sense. My argument here is fairly weak -- I couldn't comme up with a reasonable name, so I got stuck with `Poll()`, which is also the task interface runction name for polling O:-)